ProxyAgent/Runtime/Config.cs

```diff
@@ -40,33 +40,33 @@ public class Config : IConfig
         private const bool REDIRECT_HTTP_DEFAULT = true;
         private const bool STS_ENABLED_DEFAULT = true;
         private const int STS_PERIOD_DEFAULT = 2592000;
-        private const int MAX_PAYLOAD_SIZE_DEFAULT = 131072;
-        private const bool CONFIG_STATUS_DEFAULT = false;
-        private const string LOG_LEVEL_DEFAULT = "Warn";
+        private const int MAX_PAYLOAD_SIZE_DEFAULT = 102400;
```

Comment:
Where does this number come from? A comment could be helpful for when we look back on this later.

Reply:
👍